src/occwl/io.py

```diff
@@ -1,18 +1,13 @@
-from OCC.Core.STEPControl import STEPControl_Reader
 from occwl.compound import Compound
 from occwl.solid import Solid
 from occwl.face import Face
 from occwl.edge import Edge
-from occwl.geometry import geom_utils
 from OCC.Extend.DataExchange import export_shape_to_svg
-from OCC.Extend import TopologyUtils
 from OCC.Core.gp import gp_Pnt, gp_Dir
-from OCC.Core.STEPControl import STEPControl_Writer, STEPControl_AsIs
-from OCC.Core.Interface import Interface_Static_SetCVal
-from OCC.Core.IFSelect import IFSelect_RetDone
+from deprecate import deprecated
 
-from pathlib import Path
 
+@deprecated(target=None, deprecated_in="0.0.3", remove_in="0.0.5")
```

Comment:
I think this is also being used in UVStyleNet, so you will need to fix that as well